### PR TITLE
KEYCLOAK-10610 Update image for CD7

### PR DESCRIPTION
**Pull request by martin-kanis**
Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for other issues
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Review by iankko — APPROVED** [review]
LGTM

**Comment by martin-kanis** [comment]
@iankko I changed commit id reference to CD17_CR1 tag as it's available now.

**Comment by iankko** [comment]
> @iankko I changed commit id reference to CD17_CR1 tag as it's available now.

@martin-kanis Ok, great. Thanks!

File: image.yaml [hunk 1]
```diff
@@ -29,11 +29,11 @@ modules:
           - name: cct_module
             git:
                 url: https://github.com/jboss-openshift/cct_module.git
-                ref: sprint-31
+                ref: sprint-35
           - name: jboss-eap-7-image
             git:
                 url: https://github.com/jboss-container-images/jboss-eap-7-image.git
-                ref: CD16-CR1
+                ref: CD17_CR1
```
Comment (hmlnarik) [inline image.yaml]: There will be a respin of CD 17 so this will need update.

**Comment by martin-kanis** [comment]
@iankko @hmlnarik updated to CD17_CR2 and to 0.35.1 for cct_modules

**Review by hmlnarik — APPROVED** [review]

**Review by iankko — APPROVED** [review]
lgtm

**Comment by hmlnarik** [comment]
@martin-kanis Merging #88 has caused the conflicts, could you please rebase?

**Review by hmlnarik — APPROVED** [review]

**Pull request merged**